content/spin/v2/triggers.md

```diff
@@ -87,24 +87,22 @@ If you are not sure, or are not experienced, we recommend using named components
 
 In this section, we build an application that contains multiple triggers.
 
-> Note: Not all templates support the `spin add` command. In particular, at the time of writing, none of the default Redis templates support being added to existing applications. Therefore, if you want to use `spin add` to build an application with both Redis and HTTP triggers, you should first create a Redis application, then use `spin add` to add HTTP triggers, as shown below. (You won't be able to add additiona Redis triggers this way; if you need those, you'll need to set them up manually for now.)
-
 Here is an example of creating an application with both HTTP and Redis triggers:
 
 <!-- @nocpy -->
 
 ```bash
-# Start with a Redis trigger application
-$ spin new -t redis-rust trigger-example
+# Start with HTTP trigger application
```

Comment:
Good catch to update this - thanks!  However, I'd suggest following https://www.fermyon.com/blog/spin-application-structure and doing:

```
$ spin new -t http-empty trigger-example
$ cd trigger-example
$ spin add -t http-rust http-trigger-example
$ spin add -t redis-rust redis-trigger-example
```

This will give an evenly laid out directory structure instead of the HTTP code being in a top-level directory and the Redis code in a child one.  (However, it will have a knock-on effect on some of your example manifests, as the HTTP component will now have extra directories.)

Reply:
Great idea. I will update now.